### ISSUE TITLE
Mongoid 3

### ISSUE THREAD

**Issue by krasnoukhov** [flyingmachine/whoops#10]
Hey @flyingmachine, your gem looks awesome, do you have any plans on upgrade to Mongoid 3?

**Comment by flyingmachine** [comment]
Yes! As a matter of fact, I'm going to be spending a lot of quality time with Whoops over the next month, and this is one of the many things I have in store.

**Comment by krasnoukhov** [comment]
It would be nice. I have some experience with Mongoid and have some time to spend on opensource. Maybe i can help you?
Also, is there possibility to set up web interface inside existing Rails app?

**Comment by flyingmachine** [comment]
Since it's an engine, you should be able to add it to an existing Rails app. However, I don't recommend adding it to the Rails app that will be sending the logs, as an error with the Rails app might make it impossible to log anything.

I'm not sure yet whether it'd be a good idea for you to do the Mongoid changes... I'm going to be changing a lot of things, and wouldn't want to step on your toes. Thank you very much for the offer, though!
